website/docs/learning-and-resources/how-to-guides/how-to-pass-params-to-an-api.mdx

```diff
@@ -0,0 +1,77 @@
+---
+description: Learn how to pass parameters to your queries with the run() framework function.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# How to Pass Parameters to Queries
+
+This page describes how to use the run() function to pass contextual parameters to your queries.
+
+<VideoEmbed host="youtube" videoId="znaaDiQbAS8" title="How to pass parameters to an API call" caption="How to pass parameters to an API call"/>
+
+## Pass parameters in a loop
```

Comment:
This looks like a tutorial rather than a how to guide and won't help users who are just looking to see how to pass parameters on queries.  Let's discuss this in the weekly review call.